components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -202,14 +218,35 @@ The member expression dialog also supports tokens, because candidate tokens are
 You enter a token using the expression `token:<id>`.
 In order to find a token's ID, use the command in the API reference for [listing tokens]({{< relref "iam-v2-api-reference#listing-tokens" >}}).
 
+### Projects
+
+Projects are used to group and permission Chef Automate resources, ingested client-runs and compliance nodes.
+
+Projects can be managed via the Projects list under the *Settings* tab and consist of an ID, a name, and a collection of rules. Project rules are lists of conditions that describe ingested
+resources, so they are not necessary when assigning IAM resources such as teams or roles.
+
+To create a project, navigate to the Projects list under the **Settings** tab and select **Create Project**. You will need to provide a name and can optionally edit the ID. You must create a project
+before you can assign it to any resources.
+
+You can use the browser to assign projects to teams and tokens. To assign projects to a team, select a team from the _Teams_ list, then select **Details**.
+Likewise, to assign projects to a token, select a token from the API tokens list, then select **Details**.
+In either case, you can select projects from the projects dropdown to assign.
+
+Presently, projects cannot be assigned to users, and must be assigned to polices and roles using the command line.
+
 ## Removing Legacy Policies
 
-Once you've rewritten your v1 policies as v2 policies, you should remove the v1 legacy policies.
-This section walks you through deleting all of the legacy policies imported into v2, which will provide you a clean slate for exploring the IAM v2 Beta.
+This section walks you through deleting legacy policies imported into v2, which will provide you a clean slate for exploring the IAM v2 Beta.
```

Comment:
itd be really cool if we could avoid idioms like _clean slate_ and instead use plain language, this will make it easier for folks from other countries to consume our docs.